Improve networks filters in all projects query

mohammadranjbarz opened the pull request:
related to #1096

mohammadranjbarz committed:
Improve networks filters in all projects query

related to #1096

mohammadranjbarz committed:
Resolve merge conflicts

CarlosQ96 committed:
add tests for staging goerli and alfajores

CarlosQ96 reviewed (approved):
LGTM Mohammad, thanks for adding goerli and alfajores filters. I did it on my branch but you already had it here. So I just moved my tests here.

mohammadranjbarz committed:
Fix test cases for alfadores network

related to #1096

mohammadranjbarz committed:
Fix test cases for alfadores network

related to #1096

mohammadranjbarz commented:
> LGTM Mohammad, thanks for adding goerli and alfajores filters. I did it on my branch but you already had it here. So I just moved my tests here.

Thanks for bringing those test cases, I fixed them so I'll merge it now
